src/main/java/filter/expression/Qualifier.java

```diff
@@ -996,42 +1039,26 @@ private Qualifier convertMilestoneAliasQualifier(List<TurboMilestone> allMilesto
             return this;
         }
 
-        int offset;
-        int newIndex = currentIndex;
+        int offset = 0;
         if (aliasMatcher.group(2) != null && aliasMatcher.group(3) != null) {
-            offset = Integer.parseInt(aliasMatcher.group(3));
             if (aliasMatcher.group(2).equals("+")) {
-                newIndex += offset;
+                offset = Integer.parseInt(aliasMatcher.group(3));
             } else {
-                newIndex -= offset;
+                offset = -Integer.parseInt(aliasMatcher.group(3));
             }
         }
 
         // if out of milestone range, don't convert alias
```

Comment:
We can change this comment to `if offset is not within milestone alias range, no issues should be shown`.